hepcrawl/spiders/arxiv_spider.py

```diff
@@ -77,41 +86,81 @@ def parse_node(self, response, node):
         return record.load_item()
 
     def _get_authors_or_collaboration(self, node):
+        """Parse authors, affiliations; extract collaboration"""
         author_selectors = node.xpath('.//authors//author')
 
+        # take 'for the' out of the general phrases and dont use it in affiliations
+        collab_phrases = ['consortium', ' collab ', 'collaboration', ' team', 'group', ' on behalf of ', ' representing ']
+        inst_phrases = ['institute', 'university', 'department', 'center']
+
         authors = []
         collaboration = []
         for selector in author_selectors:
             author = Selector(text=selector.extract())
-            forenames = get_first(author.xpath('.//forenames//text()').extract(), default='')
-            keyname = get_first(author.xpath('.//keyname//text()').extract(), default='')
-
-            # Check if keyname is a collaboration, else append to authors
-            collab_phrases = ['consortium', 'collab', 'collaboration', 'team', 'group', 'for the']
-            collab_found = any(phrase for phrase in collab_phrases if phrase in keyname.lower())
-
-            if collab_found:
-                collaboration.append(keyname)
+            forenames = ' '.join(author.xpath('//forenames//text()').extract())
+            keyname = ' '.join(author.xpath('//keyname//text()').extract())
+            name_string = " %s %s " % (forenames, keyname)
+            affiliations = author.xpath('//affiliation//text()').extract()
+
+            # collaborations in affiliation field? Cautious with 'for the' in Inst names
+            collab_in_aff = []
+            for index, aff in enumerate(affiliations):
+                if any(phrase for phrase in collab_phrases if phrase in aff.lower()) \
+                    and not any(phrase for phrase in inst_phrases if phrase in aff.lower()):
```

Comment:
There's a dummy indentation issue here, pep8 does not like that you have only on indent level for the continuation of the if line, you should use two levels and then start just one level in, like:

```
if something \
        and soething:
    do somthing else
```
It's what's making the tests fail :(

Still getting used to the code and such, but I'll try to review the rest soon, thanks!!